File: PHP/Assignment-2/q1.php
<!-- 1. If Statement – Check Even or Odd
Objective: Write a PHP program to check whether a number is even or odd using the if statement.
Input: Any number.
Output: Print "Even" or "Odd". -->

<?php
    $number = 5; // You can change this number to test with different values
    if ($number % 2 == 0) {
        echo "Even";
    } else {
        echo "Odd";
    }
?>

File: Python/Lab-3/q1.py
# 1. Write a Python function to calculate the factorial of a number. Call the function for multiple test cases.

def factorial(n):
    if n == 0:
        return 1
    else:
        return n * factorial(n-1)

# Test cases
print(factorial(5))  # Output: 120
print(factorial(3))  # Output: 6
print(factorial(0))  # Output: 1